### PR TITLE
Add an option to translate end user errors

### PR DESCRIPTION
**Pull request by orius123**
## Related Issues

Related https://github.com/descope/etc/issues/3022 

## Description

Add an option to translate end user errors

### CODE REVIEW

File: src/components/Descope.tsx [hunk 1]
```diff
@@ -93,6 +94,12 @@ const Descope = React.forwardRef<HTMLElement, DescopeProps>(
 			};
 		}, [innerRef, onError, handleSuccess]);
 
+		useEffect(() => {
```
Comment (asafshen) [inline src/components/Descope.tsx:97]: How about adding tests ? Should be simple I think - just make sure that the wc instance has this property

**Review by asafshen — DISMISSED** [review]
Looks great 💯

**Review by nirgur — DISMISSED** [review]
👍

**Review by asafshen — APPROVED** [review]
🥇

**Comment by dorsha** [comment]
@orius123 let's communicate this to Zafran

**Pull request merged**